File: src/Enterprise/VB.NET/patterns/EnterpriseBridge.vb
Imports System
Friend Module EnterpriseBridgeExample
    Friend Function Run() As Boolean
        Dim send As Func(Of String,String,String,String)=Function(t,k,m) $"{t}>{k}:{m}"
        Return send("kafka","ALERT","disk")="kafka>ALERT:disk" AndAlso send("queue","REMINDER","backup")="queue>REMINDER:backup"
    End Function
End Module

File: src/Enterprise/C#/patterns/EnterpriseBridge.cs
namespace Genkidama.PatternExamples;
public static class EnterpriseBridgeExample { public static bool Run(){string Send(string t,string k,string m)=>$"{t}>{k}:{m}";return Send("kafka","ALERT","disk")=="kafka>ALERT:disk"&&Send("queue","REMINDER","backup")=="queue>REMINDER:backup";} }
